### PR TITLE
audit: add comment on googlecode.com https and SVN

### PR DESCRIPTION
**Pull request by vszakats**

### CODE REVIEW

**Comment by DomT4** [comment]
Presuming this is for the ` svn ` issue - There's a reason we don't enforce it: https://github.com/Homebrew/homebrew/pull/39200#issuecomment-97629674

**Comment by vszakats** [comment]
Yes, it is. Got it. What do you think of adding a reminder in comment?

**Comment by DomT4** [comment]
No objection from me if it helps stop the "What about xyz svn https" issue popping up every now and again. I have no power though, you'll need to ask one of the maintainers :smiley_cat:

**Comment by vszakats** [comment]
Changed the patch to keep the logic and add a short comment instead. Feel free to discard it. It won't be needed for long, given Google Code service being retired in a short while. There's only about 10-ish formulae affected.

File: Library/Homebrew/cmd/audit.rb [hunk 1]
```diff
@@ -381,6 +381,8 @@ def audit_urls
     end
 
     # Check for Google Code download urls, https:// is preferred
+    # Intentionally not extending this to SVN repositories due 
+    # to certficiate issues.
```
Comment (mikemcquaid) [inline Library/Homebrew/cmd/audit.rb]: `certficiate` typo :)

Comment (vszakats) [inline Library/Homebrew/cmd/audit.rb]: Thanks! corrected!

**Comment by vszakats** [comment]
This commit is probably pointless, closing it. Leaving the branch there for some days just in case.

**Comment by DomT4** [comment]
I don't know if any of the other maintainers have seen this yet - Things drop off the first page of PRs pretty quickly, and Mike isn't around at the moment.

**Comment by vszakats** [comment]
Thanks for the info Dominyk. I'm reopening for Mike to see it once he gets back.

**Pull request closed without merging**